src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -8,19 +8,15 @@
 import java.util.*;
 import java.util.function.Function;
 
-// TODO: figure out a way to include funky template size data without swamping ourselves in a sea of evidence
-// TODO: maybe count each such read as fractional evidence?
-
 /**
  * Figures out what kind of BreakpointEvidence, if any, a read represents.
  */
 public class ReadClassifier implements Function<GATKRead, Iterator<BreakpointEvidence>> {
     @VisibleForTesting static final int MIN_SOFT_CLIP_LEN = 30; // minimum length of an interesting soft clip
     @VisibleForTesting static final int MIN_INDEL_LEN = 40; // minimum length of an interesting indel
     private static final byte MIN_QUALITY = 15; // minimum acceptable quality in a soft-clip window
-    //private static final float MIN_ZISH_SCORE = -4.f;
-    //private static final float MAX_ZISH_SCORE = 6.f;
-    //private static final float CRAZY_ZISH_SCORE = 100.f;
+    private static final float MAX_ZISH_SCORE = 6.f; // maximum fragment-length "z" score for a normal fragment
+    private static final float MIN_CRAZY_ZISH_SCORE = 100.f; // "z" score that's probably associated with a mapping error
```

Comment:
Yes, the value has to be set pretty high or we get swamped.  I just felt around for a value that would produce a reasonable amount of evidence on NA12878-30X.